x-pack/plugins/apm/public/components/shared/search_bar.tsx

```diff
@@ -22,15 +27,26 @@ const SearchBarFlexGroup = styled(EuiFlexGroup)`
 interface Props {
   prepend?: React.ReactNode | string;
   showTimeComparison?: boolean;
+  showCorrelations?: boolean;
 }
 
 function getRowDirection(showColumn: boolean) {
   return showColumn ? 'column' : 'row';
 }
 
-export function SearchBar({ prepend, showTimeComparison = false }: Props) {
+export function SearchBar({
+  prepend,
+  showTimeComparison = false,
+  showCorrelations = false,
+}: Props) {
+  const { uiSettings } = useApmPluginContext().core;
+  const license = useLicenseContext();
+  const hasCorrelationsAccess =
+    uiSettings.get(enableCorrelations) && isActivePlatinumLicense(license);
```

Comment:
Are we going to remove this setting? What advantages are there to leave it in?

Reply:
removed